packages/app-form-builder/src/admin/components/FormEditor/Tabs/EditTab/FormStep/FormStep.tsx

```diff
@@ -194,9 +234,25 @@ export const FormStep = ({
                                     </div>
                                     <Horizontal
                                         onDrop={item => {
-                                            handleDropField(item, {
-                                                row: index,
-                                                index: null
+                                            handleDrop({
+                                                target: {
+                                                    type: item.ui,
+                                                    id: item.id,
+                                                    name: item.name
+                                                },
+                                                source: {
+                                                    containerId: item?.container?.id,
+                                                    containerType: item?.container?.type,
+                                                    position: item.pos
+                                                },
+                                                destination: {
+                                                    containerId: formStep.id,
+                                                    containerType: "step",
```

Comment:
This will be changed!
We are gonna get data for `destination` from `dropTarget`
https://github.com/webiny/webiny-js/pull/3574/files#diff-33b595e920d5882dbdc98d6382274a874914bbfc92ff1b4c27df2ff8a711e48cR109